app/models/concerns/raceable.rb

```diff
@@ -15,8 +15,14 @@ module Raceable
 
     # Return all current races, as defined by any raceable less than an hour old or has 2 or more entrants
     def self.current
-      where('updated_at > ?', 1.hour.ago)
-        .or(where(id: Entrant.having('count(*) > 1').group(:raceable_id).select(:raceable_id)))
+      case name # self.name refers to the class including this concern
```

Comment:
Was this doing the thing where it doesn't know which table's column to compare against?

Reply:
Yeah, `updated_at` was an ambiguous column reference when I tried to do `@game.races.current`.